hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4955,6 +4955,19 @@
   </description>
```

Comment:
for `dfs.journalnode.edit-cache-size.bytes`, we also need to update the documentation:

1. remove the default value, since there is none. mention that the size defaults to the value of the `.fraction` config
2. mention that it takes precedence over the `.fraction` config, so if both are set, then `.bytes` is used

Reply:
Thanks for your thoughtful advice.